### PR TITLE
Update to sdk 1.1.0

### PR DESCRIPTION
**Pull request by MichaelSimons**
Related #364 

Note:  Latest SDK hasn't picked up the latest runtime versions yet therefore the runtime images won't be updated at this time.

### CODE REVIEW

File: update-dependencies/CliDependencyHelper.cs [hunk 1]
```diff
@@ -35,11 +49,12 @@ public static string GetSharedFrameworkVersion(string cliVersion)
...
         }
 
-        private static string GetCommitHash(string cliVersion)
+        private string GetCommitHash()
         {
-            using (ZipArchive archive = DownloadCliInstaller(cliVersion).Result)
+            using (ZipArchive archive = DownloadCliInstaller().Result)
             {
-                ZipArchiveEntry versionTxtEntry = archive.GetEntry($"sdk\\{cliVersion}\\.version");
+                string zipEntryName = CliMajorVersion == 1 ? $"sdk/{CliVersion}/.version" : $"sdk\\{CliVersion}\\.version";
```
Comment (crummel) [inline update-dependencies/CliDependencyHelper.cs]: I think you should be able to use "sdk/{CliVersion}/.version" in all cases and ZipArchive will handle it (this is the behavior I saw on the symbol signing stuff at least).

File: update-dependencies/Program.cs [hunk 2]
```diff
@@ -117,16 +118,33 @@ private static Task CreatePullRequest(DependencyUpdateResults updateResults)
...
             string majorMinorVersion = s_config.CliReleasePrefix.Substring(0, s_config.CliReleasePrefix.LastIndexOf('.'));
-            string searchFolder = Path.Combine(s_repoRoot, majorMinorVersion);
-            string[] dockerfiles = Directory.GetFiles(searchFolder, "Dockerfile", SearchOption.AllDirectories);
+            string[] dockerfiles = GetDockerfiles(majorMinorVersion);
             Trace.TraceInformation("Updating the following Dockerfiles:");
             Trace.TraceInformation($"{string.Join(Environment.NewLine, dockerfiles)}");
-            return dockerfiles.Select(path => CreateDockerfileEnvUpdater(path, "DOTNET_SDK_VERSION", CliBuildInfoName))
-                .Concat(dockerfiles.Select(path => CreateDockerfileEnvUpdater(path, "DOTNET_VERSION", SharedFrameworkBuildInfoName)))
-                .Concat(dockerfiles.Select(path => new DockerfileShaUpdater(path)));
+            IEnumerable<IDependencyUpdater> updators = dockerfiles
```
Comment (crummel) [inline update-dependencies/Program.cs]: Spelling: "updaters".

File: 1.0/sdk/jessie/Dockerfile [hunk 3]
```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 # Install .NET Core SDK
-ENV DOTNET_SDK_VERSION 1.0.4
+ENV DOTNET_SDK_VERSION 1.1.0-preview1-005124
```
Comment (MichaelSimons) [inline 1.0/sdk/jessie/Dockerfile]: The .NET Core docker versioning is in terms of the Runtime.  Think of it this way, if you want to build an app that targets 1.0, then this is the SDK image you should use to build your app.

File: update-dependencies/Program.cs (hunk 2 above)
Comment (MichaelSimons) [inline update-dependencies/Program.cs]: Fixed.

File: update-dependencies/CliDependencyHelper.cs (hunk 1 above)
Comment (MichaelSimons) [inline update-dependencies/CliDependencyHelper.cs]: Looking at the [ZipArchive.GetEntry implementation](https://github.com/dotnet/corefx/blob/2f554b13fe67c8cf9b37c70a711de1ba5575638c/src/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs#L261), a simple Dictionary.TryGetValue is used and no transformation is performed on the FullName.  This leads me to conclude the path syntax must match the way the zip was constructed.

**Pull request merged**